### PR TITLE
Added `verbatim` argument to `run_` and `post_circuit`

### PR DESCRIPTION
**Pull request by BrunoLiegiBastonLiegi**
As per title, allows for verbatim (non transpiled) execution. 
This is related to qiboteam/qibo-webapp#155

### CODE REVIEW

File: src/qibo_client/qibo_client.py [hunk 1]
```diff
@@ -68,7 +68,8 @@ def run_circuit(
         self,
         circuit: qibo.Circuit,
         device: str,
-        nshots: int = None,
+        nshots: T.Optional[int] = None,
+        verbatim: T.Optional[bool] = False,
```
Comment (marcorossi5) [inline src/qibo_client/qibo_client.py:72]:
```suggestion
        verbatim: bool = False,
```
Optional means that the default value is `None`

File: src/qibo_client/qibo_client.py [hunk 2]
```diff
@@ -108,7 +111,8 @@ def _post_circuit(
...
```
Comment (marcorossi5) [inline src/qibo_client/qibo_client.py:115]:
```suggestion
        verbatim: bool = False,
```

**Review by marcorossi5 — CHANGES_REQUESTED** [review]
Ok by me. When qibo releases the automatic transpiler, we should change the `qibo-daemon` too

**Comment by BrunoLiegiBastonLiegi** [comment]
> Ok by me. When qibo releases the automatic transpiler, we should change the `qibo-daemon` too

it should be done already qiboteam/qibo-daemon#48

**Comment by scarrazza** [comment]
Thanks @BrunoLiegiBastonLiegi, did you check if this is working fine with the PR for the daemon? If so, we can merge the daemon and deploy, re-test this and merge.

**Pull request merged**